README.md

```diff
@@ -61,3 +61,25 @@ To extract the policy for a given state, simply call the action function:
 ```julia
 a = action(polciy, s) # returns the optimal action for state s
 ```
+
+## Parallel Value Iteration 
+
+Value iteration can be speeded up significantly by computing the value asynchronously in parallel. This package provides an parallel implementation of the Gauss-Seidel Value Iteration algorithm. 
+
+To enable using several CPUs with julia there are two options: 
+- Start julia with `julia -p 8` (to use 8 procs)
+- write `addprocs(7)` in your script (to add 7 procs in addition to the main, 8 procs available in total)
+
+Here is an example of `ParallelValueIterationSolver` applied to a large GridWorld problem:
+```julia
+addprocs(7) # or start julia with julia -p 8
+@everywhere using POMDPs, POMDPModels # do not forget 
+@everywhere using DiscreteValueIteration
```

Comment:
I don't think these two `@everywhere`s should be necessary. `using` loads the module on every process, but only brings  it into scope on one process.